db/seeds/support/data_migrator.rb

```diff
@@ -110,6 +110,46 @@ def role_attrs_from_function(row)
       ]
     end
 
+    def subscriptions_attrs(row)
+      person = person_from_row(row)
+
+      attrs = []
+
+      if row[:no_advertising] == 'f'
+        attrs << {
+          subscriber_type: 'Person',
+          subscriber_id: person.id,
+          mailing_list_id: dachverband_mailing_list(:advertising).id
+        }
+      end
+
+      [:newsletter_dachverband_subscriber,
+       :bulletin_subscriber].each do |key|
+         if remove_lov_prefix(row[key]) == 'Yes'
+           attrs << {
+             subscriber_type: 'Person',
+             subscriber_id: person.id,
+             mailing_list_id: dachverband_mailing_list(key).id
+           }
+         end
+      end
+
+      [:versand_subscriber,
+       :newsletter_section_subscriber].each do |key|
+         if remove_lov_prefix(row[key]) == 'Yes'
+           attrs += person.groups.map do |group|
+             {
+               subscriber_type: 'Person',
+               subscriber_id: person.id,
+               mailing_list_id: versand_mailing_list(key, group).id
+             }
+           end
+         end
+       end
+
+      attrs.flatten.reject { |subscription_attrs| Subscription.exists?(subscription_attrs) }
```

Comment:
das soll doch nachher via Upsert in die DB kommen oder? Wenn dafür kein unique-Index existiert, dann ist jetzt wohl der Zeitpunkt, ihn einzubauen. Damit könnte dann das `reject` hier entfallen.